packages/lavapack/package.json

```diff
@@ -14,7 +14,7 @@
     "combine-source-map": "^0.8.0",
     "convert-source-map": "^2.0.0",
     "json-stable-stringify": "^1.0.2",
-    "lavamoat-core": "^14.2.1",
+    "lavamoat-core": "^14.2.2",
```

Comment:
lavapack's build script takes SES from core and builds it into the runtime. Let's make sure we're not overlooking it here before we release.